cmd/create/cluster/cmd.go

```diff
@@ -871,27 +899,39 @@ func run(cmd *cobra.Command, _ []string) {
 		args.ec2MetadataHttpTokens = string(v1.Ec2MetadataHttpTokensOptional)
 	}
 
+	// Errors when users elects for cluster admin via flags and elects for hosted control plane via interactive prompt"
+	if isHostedCP && (createAdminUser || createAdminPassword != "") {
```

Comment:
better to check this as early as possible

Reply:
I have another check that occurs earlier that checks for the combination of 3 flags 
https://github.com/davidleerh/rosa/blob/OCM-3831-ADMIN/cmd/create/cluster/cmd.go#L783
`--create-admin-user --create-admin-password --hosted-cp`
![image](https://github.com/openshift/rosa/assets/118839428/efcc0e57-2048-45e9-9056-1a5975a318a8)


Condition on line 903 is a separate check that occurs if the user supplies flags related to cluster admin, but 
hosted-cp is selected during interactive prompt.

![image](https://github.com/openshift/rosa/assets/118839428/54805f24-5a74-45fb-8b3c-52f5b4839ed5)